sdks/java/core/src/main/java/org/apache/beam/sdk/options/GoogleApiDebugOptions.java

```diff
@@ -40,7 +40,7 @@
    * to Google, specifically to the Google Cloud Dataflow team.
    */
   @Description("This option enables tracing of API calls to Google services used within the "
-      + "Dataflow SDK. Values are expected in JSON format {\"ApiName\":\"TraceDestination\",...} "
+      + "Apache Beam SDK. Values are expected in JSON format {\"ApiName\":\"TraceDestination\",...} "
```

Comment:
There is a lint error here:
GoogleApiDebugOptions.java:43: Line is longer than 100 characters (found 101). [LineLength]

perhaps move the empty space at the end of the line to the next line.